styles.css

```diff
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: Playfair-Display-Font;
+  src: url(fonts/PlayfairDisplay-Regular.ttf);
+}
+
+@font-face {
+  font-family: Poppins;
+  src: url(fonts/Poppins-Regular.ttf);
+}
+
+body {
+  margin-left: 0;
+  margin-top: 0;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+#toolbar {
+  height: 8%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#seperator01 {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 10px;
+  padding-top: 5em;
+}
+
+.headline {
+  margin-left: 9%;
+  display: flex;
+  width: 83%;
+  height: 40%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}

```

Comment:
- [ ] I think it would be better as suggested by the previous code reviewer, If you notice, there are still some variations in ``` font, margin,``` and ```spacing``` to the Figma design. Kindly ensure that the design is professionally following the [Figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095&t=5Ghzz5JyoY1t2OLY-0) 😎
   - I think it would be better to match mostly the ```font-size``` and ```margin``` of your elements to replicate the Figma design,
## Figma design
![image](https://user-images.githubusercontent.com/102757126/212062460-f64a20d0-f7c9-4c73-b5a6-c984a885cec3.png) 
## Your design
![image](https://user-images.githubusercontent.com/102757126/212062776-5f36ffd3-8c2c-403a-9706-4cef3e57c9f9.png)